index.js

```diff
@@ -33,6 +33,18 @@ function normalize(type, value) {
   return result;
 }
 
+function conforms(type, value) {
+  if (typeof type === 'string') {
+    return typeof value === type;
+  }
+  if (typeof type === 'function') {
+    return Boolean(type(value));
+  }
+  return type.some(function(type) {
```

Comment:
do we need an array test around this so it doesn't throw strange errors `some is not a function`?